gather-scheduler.js

```diff
@@ -36,33 +38,68 @@ class GatherScheduler {
     }
 
     return driver.connect()
+
+      // Enable emulation.
+      .then(_ => {
+        if (emulateMobileDevice) {
+          return driver.beginEmulation();
+        }
+
+        return Promise.resolve();
+      })
+
+      // Clean all caches and force updates to any Service Workers.
+      .then(_ => driver.cleanCaches())
```

Comment:
Is this cleaning the HTTP cache or the caches via the caches API?

If the later, you'd probably want the service worker to go through the full install lifecycle again if it hasn't installed which you'd do by unregistering existing sw and get it to re-register.